tests/foreman/cli/test_docker.py

```diff
@@ -224,33 +271,33 @@ def test_update_docker_repo_url(self, name):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        new_url = gen_url(scheme='https')
```

Comment:
Is that 'https' scheme required here? Just want to avoid redundant logic that can confuse people